src/Select.js

```diff
@@ -236,12 +244,22 @@ type MenuOptions = {
 };
 
 type State = {
```

Comment:
Internally we now have two new pieces of state information. 
* ariaLiveContext: (a string value holding instructions for assistive tech based on the context of the user (in menu, focused on value, in input) 
* ariaLiveSelection: a string value holding instructions for assistive tech based on value events that occur (select-option, deselect-option, pop-value etc.)